hazelcast/src/test/java/com/hazelcast/jet/core/JobTimeoutTest.java

```diff
@@ -88,10 +88,10 @@ public void when_jobIsResumedAndExceedsTimeout_jobIsCancelled() {
         final JobConfig jobConfig = new JobConfig().setTimeoutMillis(1000L);
         final Job job = hz.getJet().newJob(dag, jobConfig);
 
-        assertJobStatusEventually(job, JobStatus.RUNNING, 1);
+        assertJobStatusEventually(job, JobStatus.RUNNING);
         job.suspend();
 
-        assertJobStatusEventually(job, JobStatus.SUSPENDED, 1);
+        assertJobStatusEventually(job, JobStatus.SUSPENDED);
```

Comment:
The test below also uses 1s timeout, let's update that as well.

Reply:
ok